### PR TITLE
Assignment4

### PR DESCRIPTION
**Pull request by vDubs93**
Assignment 4 complete.  Data displayed is from openweathermap, including 16 days of forecast weather data.

### CODE REVIEW

File: app/src/main/java/com/example/weatherstation/Api.kt [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import retrofit2.http.Query
+
+interface Api {
+    @GET("weather")
+    fun getCurrentConditions(
+        @Query("zip") zip: String,
+        @Query("units") units: String = "Imperial",
+        @Query("appid") appid: String = "6d3cbb90efe05909395139d822e2357b"
+    ) : Call<CurrentConditions>
+    @GET("daily")
```
Comment (benjamincassidy) [inline app/src/main/java/com/example/weatherstation/Api.kt]: should be `@GET("forecast/daily")`.

File: app/src/main/java/com/example/weatherstation/ForecastActivity.kt [hunk 2]
```diff
@@ -4,39 +4,59 @@ import androidx.appcompat.app.AppCompatActivity
...
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_forecast)
 
+        val moshi = Moshi.Builder()
+            .add(KotlinJsonAdapterFactory())
+            .build()
+
+        val retrofit = Retrofit.Builder()
+            .baseUrl("https://pro.openweathermap.org/data/2.5/forecast/")
```
Comment (benjamincassidy) [inline app/src/main/java/com/example/weatherstation/ForecastActivity.kt]: should be `.baseUrl("https://pro.openweathermap.org/data/2.5/forecast/")`.

Comment (vDubs93) [reply app/src/main/java/com/example/weatherstation/ForecastActivity.kt]: Did you mean `.baseUrl("https://pro.openweathermap.org/data/2.5/")` ?

File: app/src/main/java/com/example/weatherstation/Api.kt (hunk 1 above)
Comment (vDubs93) [reply app/src/main/java/com/example/weatherstation/Api.kt]: Fixed and pushed.

File: app/src/main/java/com/example/weatherstation/ForecastActivity.kt (hunk 2 above)
Comment (vDubs93) [reply app/src/main/java/com/example/weatherstation/ForecastActivity.kt]: Either way, fixed and pushed.

**Pull request merged**